core/src/com/group66/game/helpers/ScoreKeeper.java

```diff
@@ -0,0 +1,29 @@
+package com.group66.game.helpers;
+
+public class ScoreKeeper {
+	
+	public int currentScore;
+	
+
+	public ScoreKeeper(int currentScore) {
+		this.currentScore = 0;
+	}
+	
+
+	// getting the current score
+	public int getCurrentScore() {
+		return currentScore;
+	}
+
+	//calculating the new score
+	public void setCurrentScore(int poppingBalls, int hangingBalls) {
+		this.currentScore = this.currentScore + (poppingBalls + (2^hangingBalls))* 10; //CHECK is this correct?
```

Comment:
I think its is correct, the original game had a max score for 17 or more dropped balls, but lets just keep it like this for now, as there where also time based score bonuses. (so keep it and remove the CHECK comment)

"Popped" bubbles (that is, bubbles of the same color which disappear) are worth 10 points each. However, dropped bubbles (that is, bubbles that were hanging from popped bubbles), are worth far more: one dropped bubble scores 20 points; two score 40; three score 80. This figure continues doubling for each bubble dropped, up to 17 or more bubbles which scores 1,310,720 points" -Wikipedia https://en.wikipedia.org/wiki/Puzzle_Bobble#Scoring_system